changelog.yaml

```diff
@@ -52,6 +52,18 @@
   date: TBD
   changes:
 
+    - type: enhancement
+      impact: patch
+      title: Add ignore label
+      description: |-
+        For tests it is required to create/update Steward CROs, but avoid that
```

Comment:
```suggestion
        For tests it is required to create/update Steward CRDs, but avoid that
```

Reply:
"CROs" (custom resource objects) is correct. We don't modify the CRDs.